apps/dh/api-dh/source/DataHub.WebApi/Modules/Processes/MoveIn/CustomerIdentificationInput.cs

```diff
@@ -1,6 +1,6 @@
-// Copyright 2020 Energinet DataHub A/S
+﻿// Copyright 2020 Energinet DataHub A/S
 //
-// Licensed under the Apache License, Version 2.0 (the "License2");
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
nitpick: let's keep it as before

```suggestion
// Licensed under the Apache License, Version 2.0 (the "License2");
```